package.json

```diff
@@ -9,5 +9,11 @@
   "license": "Apache-2.0",
   "files": [
     "dist/**/*"
-  ]
+  ],
+  "scripts": {
+    "generate": "bikeshed-to-ts --in ./gpuweb/spec/index.bs --out ./dist/generated.d.ts --forceGlobal --nominal"
```

Comment:
We don't need to include this when publishing, so let's put it outside `dist`. How about `./generated/index.d.ts`?

To make sure the directory is there, you can put the gitignore in `./generated/.gitignore` instead.